### DISCUSSION TITLE
Best way to store multiple DNS alias of an IP

### DISCUSSION THREAD

**Discussion by smutel** [netbox-community/netbox#9369] [Q&A]
Hello,

What is for you the best way to store multiple DNS alias of an IP ?
We can set the main DNS entry in the field _DNS Name_ but I don't see another field for aliases.

Thanks.

**Comment by jeremystretch** [comment]
NetBox does not manage DNS records natively. You may be interested in the [netbox-bgp plugin](https://github.com/auroraresearchlab/netbox-dns).
